content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Unified Service Tagging
+kind: documentation
+aliases:
+- /tagging/unified_service_tagging
+further_reading:
+- link: "/tagging/using_tags"
+  tag: "Documentation"
+  text: "Learn how to use tags in the Datadog UI"
+- link: "/blog/autodiscovery-docker-monitoring"
+  tag: "Blog"
+  text: "Learn more about Autodiscovery"
+---
+
+## Overview
+Unified service tagging ties Datadog telemetry together through the use of three [reserved tags][1]: `env`, `service`, and `version`. These tags allow you to correlate related logs and traces by version of a service, trace a spike in CPU usage to corresponding areas of a service, and organize infrastructure more effectively.
+
+The [environment variables][2] `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` form a single point of configuration for all telemetry emitted directly from your service's runtime: [traces][3], [logs][4], and [StatsD metrics][5]. These variables are specific to the runtime of your service. They allow you to define core tagging configuration within your application, rather than across your application and the Agent.
+
+When spans are configured with `DD_VERSION`, the tracer will add version to all spans that fall under the service that "owns" the tracer (generally DD_SERVICE). This means that if your service creates spans with the name of an external service, those spans will not receive version as a tag. As long as version is present in spans, it will be added to trace metrics generated from those spans. The version can be added manually in-code or automatically by the tracer.
```

Comment:
What are spans?

We should not target product specific elements into an overview and assume that users do not know Datadog specific terminology.
Let's make sure that we are as explicit as possible and only introduce specific notion when required.

I would move this into a `Set the version` section